src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -149,14 +143,14 @@ public Arm() {
         for (int i = 0; i < motors.length; i++) {
             // create the motor
             motors[i] = new TalonFX(ArmConstants.MOTOR_IDS[i]);
-            // TODO: why is this done here? The slave should follow when neutral mode is set below.
             motors[i].setNeutralMode(NeutralModeValue.Brake);
 
             // i==0 is the master; the others are slaves
             if (i > 0) {
                 // set slave mode
                 // the master is MOTOR_IDS[0]
                 // invert master for ids 2 and 3
+                
                 motors[i].setControl(new Follower(ArmConstants.MOTOR_IDS[0], (i >= motors.length / 2)));
```

Comment:
No blank line. The "invert master for ids 2 and 3" relates to the new Follower().